SixTrack/sixtrack.s

```diff
@@ -36839,10 +37253,13 @@ C     Convert r(1), r(2) from U(0,1) -> rvec0 as Gaussian with cutoff mcut (#sig
             endif
 
 !----------count 56
-            goto(20,40,740,500,500,500,500,500,500,500,60,80,100,120,   &
-     &140,160,180,200,220,240,290,310,330,350,370,390,410,430,          &
-     &450,470,490,260,520,540,560,580,600,620,640,660,680,700,720       &
-     &,730,748,500,500,500,500,500,745,746,751,752,753,754),ktrack(i)
+            goto( 20, 40,740,500,500,500,500,500,500,500,!0
```

Comment:
Thanks a TON for cleaning up these a bit! It's been on my mental todo-list for... a while. But! Note that (1) Fortran counts from 1, and other places (thin6d/collimation etc.) I always labeled them as !10, !20, etc. where the !10 indicates that the LAST label on that line (here 500) corresponds to ktrack=10. When I think about it - maybe `!1-10` would be most clear?